File: 250620/코드네임/code-name.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        // Please write your code here.

        int minScore = 101;
        String agentName = "";
        Agent[] agents = new Agent[5];
        for (int i = 0; i < 5; i++) {
            String codeName = sc.next();
            int score = sc.nextInt();

            agents[i] = new Agent(codeName, score);
            minScore = Math.min(minScore, agents[i].score);
            if (minScore == agents[i].score) {
                agentName = agents[i].codeName;
            }
        }
        System.out.print(agentName + " " + minScore);
    }

    
}

class Agent {
    String codeName;
    int score;

    Agent(String codeName, int score) {
        this.codeName = codeName;
        this.score = score;
    }
}

File: 250923/코드네임/code-name.py
MAX_N = 5

class M:
    def __init__(self,codename,score):
        self.codename=codename
        self.score=score

codenames = []
scores = []
mans = []
for i in range(MAX_N):
    codename, score = input().split()
    codenames.append(codename)
    scores.append(int(score))

    mans.append(M(codenames[i],scores[i]))

minScore = min(man.score for man in mans)
minScoreMan = [man.codename for man in mans if man.score == minScore]

print(minScoreMan[0],minScore)

# Please write your code here.


